docs/documentation/recipes.md

```diff
@@ -238,7 +238,9 @@ function Makie.plot!(
             push!(bar_tos[], s.close)
         end
         append!(colors[], [x.close > x.open for x in stockvalues])
-        colors[] = colors[]
+        notify(linesegs)
+        notify(colors)
+        return
```

Comment:
Before this worked without `notify(linesegs)` because Makie did not copy the Point2f array. Now/currently it converts to Float64, so it does copy and needs an Observable update